velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -728,6 +728,38 @@ struct ArrayRemoveNullFunctionString {
   }
 };
 
+template <typename T>
+struct ArrayNGramFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T)
+
+  void call(
+      out_type<Array<Array<Generic<T1>>>>& out,
+      const arg_type<Array<Generic<T1>>>& input,
+      int64_t size) {
```

Comment:
Nit : Use variable name "n" instead of size since that denotes what it is more clearly.